Header header
float32 frequency_ch0
float32 frequency_ch1
int32 rpm_ch0
int32 rpm_ch1
